WTWSMS/decisions/WtWSMS_religious_decisions.txt

```diff
@@ -429,17 +431,36 @@ decisions = {
 		
 		effect = {
 			give_nickname = nick_with_immortal_soul
-			activate_title = { title = d_priscillianism status = yes }
+			set_global_flag = veggiepriest
+			activate_title = { title = k_priscillianism status = yes }
 			hidden_tooltip = {
-				create_random_priest = {
+				create_character = {
+					name = "Priscillian"
 					dynasty = none
-					random_traits = yes
+					trait = humble
+					trait = vegetarian
+					trait = zealous
+					trait = mystic
+					race = egyptian_arabic
+					religion = priscillianism
+					culture = ROOT
+					female = no
+					attributes = {
+						martial = 8
+						intrigue = 10
+						learning = 10
+					}
+					historical = yes
 				}
 				new_character = {
-					d_priscillianism = {
+					k_priscillianism = {
```

Comment:
The Priscillian Patriarch was essentially considered a "true Pope" and thus matches what the Bishop of Rome ultimately becomes. The Great Schism is not fully implemented yet, but I just used foresight. Also, don't change it back to the requirement that the Priscillian faith be dominant. That is unrealistic: the Priscillians were constantly grappling against the Nicenes. If they seized power, they would immediately declare Avila to be their Holy See!